src/main/java/org/testeditor/fixture/core/TestRunReporter.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.fixture.core;
+
+/**
+ * Used for reporting actions during test execution. Each action can be
+ * described unambiguously by a combination of SemanticUnit x Position x Message
+ * 
+ * Listeners can be registered to be informed about actions. The can either
+ * register for SemanticUnit x Position, Position, SemanticUnit or for all
+ * actions.
+ */
+public interface TestRunReporter {
+	enum SemanticUnit {
+		TEST, SPECIFICATION, COMPONENT, STEP
+	}; // order is relevant!
+
+	enum Position {

```

Comment:
`Position` describes not what I expected. I don't have a good word either but maybe somthing like `Transition` or `Action` or simply `Type`?